scripts/generate_pinval/generate_pinval.py

```diff
@@ -81,24 +78,16 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument(
         "--township",
         help=(
-            "Restrict triad mode to a single Cook County township code "
-            "(two-digit string, e.g. 01, 23). Ignored unless --triad is used."
+            "Restrict all-PIN mode to a single Cook County township code "
+            "(two-digit string, e.g. 01, 23). Ignored if --pin is used."
         ),
     )
 
     args = parser.parse_args()
 
-    # ── Validation ────────────────────────────────────────────────────────────
     if args.pin == [""]:
-        # Cast empty string to null
-        args.pin = None
-
-    provided_pin = bool(args.pin)
-    provided_triad = bool(args.triad)
-
-    if provided_pin == provided_triad:
-        # Either both supplied *or* neither supplied → invalid
-        parser.error("Exactly one of --triad or --pin must be provided, but not both.")
+        # Remove empty string
+        args.pin = []
```

Comment:
I think empty list is a more intuitive "empty" value for this argument than a null.